### PR TITLE
Bug/misc

### PR DESCRIPTION
**Pull request by ayyubibrahimi**
-  **Improved readability for NOPD:** I edited the format of NOPD's allegation column to `rule 04: performance of duty; paragraph 08: hours of duty` from my previous edit of `rule 04: performance of duty; paragraph 08 - hours of duty`. Thoughts?

### CODE REVIEW

**Comment by pckhoi** [comment]
is there an `allegation_create_year` column in the schema already?

**Comment by ayyubibrahimi** [comment]
There is an `allegation_create` column in the schema located in datavalid.ylm

File: lib/events.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 COMPLAINT_INCIDENT = "complaint_incident"
 COMPLAINT_RECEIVE = "complaint_receive"
-CHARGES_CREATE = "charges_create"
+ALLEGATION_CREATE = "allegation_create"
```
Comment (pckhoi) [inline lib/events.py:26]: Please modify the event kind options list in `datavalid.yml` too.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**